git/ref_filter_test.go

```diff
@@ -0,0 +1,101 @@
+package git_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/github/git-sizer/git"
+)
+
+func TestPrefixFilter(t *testing.T) {
+	t.Parallel()
+
+	for _, p := range []struct {
+		prefix   string
+		refname  string
+		expected bool
+	}{
+		{"refs/heads", "refs/heads/master", true},
+		{"refs/heads", "refs/tags/master", false},
+		{"refs/heads", "refs/he", false},
+		{"refs/heads", "refs/headstrong", false},
+		{"refs/heads", "refs/tags/refs/heads", false},
+		{"refs/stash", "refs/stash", true},
+		{"refs/stash", "refs/stashy", false},
+		{"refs/remotes", "refs/remotes/origin/master", true},
```

Comment:
You don't appear to have any tests covering prefixes ending with `/`.  Is it worth adding a couple of tests to assert the documented behaviour?

```
 // * Prefix "refs/foo/" matches "refs/foo/bar" but not "refs/foo" or
 //   "refs/foobar".
```

Reply:
Good point. Done :ballot_box_with_check: